src/components/base/anchor/Anchor.js

```diff
@@ -112,8 +112,27 @@ function Menu({
   const isScrollingClass =
     isSticky && isScrolling ? `scrolling ${styles.isScrolling}` : "";
 
+  const handleClick = (e, id, section) => {
```

Comment:
function

Reply:
skal ikke bruge _this context, så ingen grund til at bruge es5 function

Reply:
Du kan godt kalde det en "regular function", for jeg mener da at Javascript havde funktioner før es5, eller hvad?
Og de 2 er stadigvæk ligestillede, og vi bruger generelt functions i bibdk

Problemet med `const` er, at det kræver dobbelt søgning når jeg skal finde deklarationen med "Search everywhere", fordi arrow-functions ikke kan erstatte normal funktion... Og det glemmer man altså...

Af interesse: Hvorfor foretrækker du egentlig arrow-functions til navngivede funktioner, for de er jo altid anonyme?

Reply:
ja javascript havde funktioner før es5.. arrow funktions blev released i es6, derfor er det en ret normal hendvisning at kalde dem es5 funktioner. (aka. de gamle)

- function er noget man bruger i en class, da den binder "this", "super" osv. I React bruger vi kun funktions programmering.
- Den er ikke anonym hvis den er named? her `const handleClick`. Så kan man også bruge den anonymt i fx en _.map(() = {}) - ligesom _.map(function() {}
- "=>" er kortere at skrive end "function". 

Hvordan ved du om du har brugt "function()" når du søger efter den? der er pt over 2000 arrow functions i projektet.

Reply:
bare lige for at være klar i spyttet. Jeg argumentere ikke for at vi refaktorer alt væk - det fint nok de bliver brugt. 
Grunden til jeg skriver den her er mest af alt min muscle memory der defaulter til arrow